### DISCUSSION TITLE
Building a modified shim

### DISCUSSION THREAD

**Discussion by Darcii** [3kh0/ext-remover#469] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello I downloaded my grunt.bin and hit start, and selected my shim however nothing happens after I select it. Im on a chromebook not sure if that can cause issues. It doesnt do anything so im asking for help.

**Comment by Darcii** [comment, marked as answer]
Bro I stg I switched back over to the tab after trying this 10 times NOW IT WORKS AFTER I SWEAR

**Comment by npulsipher4** [comment]
YAY!!! Where did you download the shim from?

**Reply by Darcii** [reply]
> YAY!!! Where did you download the shim from?

Its on the website bro ....

**Comment by npulsipher4** [comment]
Yeah it was blocked for me so... yeah.

**Comment by ehunt34** [comment]
Do it on Linux. The web version is unreliable at best.

**Reply by Darcii** [reply]
> Do it on Linux. The web version is unreliable at best.

How so? It worked fine for me. Its just a CLI right?

**Comment by vernal-ria0** [comment]
how did you get it to work or how long did it take to work
